tasks/main.yml

```diff
@@ -157,37 +165,98 @@
     mode=0755
   become_user: "{{ project_user }}"
 
+- name: configure celery and celery beat
+  template: src=celery.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-celery-{{ item.name }}.conf
+            owner=root
+            group=root
+            mode=0600
+  with_items:
+    - { name: "default", command: "worker", flags: "--loglevel=INFO" }
+    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile={{ root_dir }}/celerybeat.pid --loglevel=INFO" }
+  notify:
+    - restart celery
+    - restart celery-beat
+
+- name: ensure celery is running
+  supervisorctl: name={{ project_name }}-celery-default state=started
+
+- name: ensure celery-beat is running
+  supervisorctl: name={{ project_name }}-celery-beat state=started
+
+- name: add nodejs repo
+  apt_repository: repo="deb https://deb.nodesource.com/node_4.x {{ ansible_distribution_release }} main"
+                  state=present
+                  filename='nodesource'
```

Comment:
Error: `TASK [tequila-django : add nodejs repo] ****************************************
fatal: [server1]: FAILED! => {"changed": false, "failed": true, "msg": "unsupported parameter for module: filename"}
`